src/app/test/db/mongo/util_test.py

```diff
@@ -315,19 +285,11 @@ def setup_class(cls):
     def teardown_method(self):
         beer_garden.db.mongo.models.Role.drop_collection()
 
-    def config_get_value(self, config_name):
```

Comment:
These got moved into mocks outside of the class so I could use them elsewhere.  That's why a bunch of unrelated tests got touched here.